sonic_platform_base/sonic_xcvr/api/public/cmis.py

```diff
@@ -1084,6 +1084,14 @@ def set_lpmode(self, lpmode):
                 lpmode_val = lpmode_val | (1 << CmisApi.LowPwrRequestSW)
                 self.xcvr_eeprom.write(consts.MODULE_LEVEL_CONTROL, lpmode_val)
                 time.sleep(0.1)
+                start_time = time.time()
+                pwroff_duration = self.get_module_pwr_down_duration()/1000
+                # Loop until the power-off duration has elapsed
+                while (time.time() - start_time) < pwroff_duration:
+                    if self.get_lpmode():
+                        return self.get_lpmode()
+                    # Sleep for a short interval before the next check
+                    time.sleep(0.1)
```

Comment:
@AnoopKamath Can we define a constant instead of using 0.1 here?

Reply:
done